addons/phone_validation/models/phone_blacklist.py

```diff
@@ -78,28 +89,27 @@ def sanitize_number(arg):
         return super()._search(domain, offset, limit, order, access_rights_uid)
 
     def add(self, number, message=None):
-        sanitized = phone_validation.phone_sanitize_numbers_w_record([number], self.env.user)[number]['sanitized']
```

Comment:
Not that 'add' could receive a non sanitized number, which should be sanitized and propagated to _add (which checks for existing numbers). Removing it could break some valid use cases I think ?

Reply:
yup, previously I had sanitation done in create method, but here I see I can't really do that, since `_add` needs them sanitaized